mmdet/models/dense_heads/nasfcos_head.py

```diff
@@ -0,0 +1,89 @@
+import copy
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, Scale, bias_init_with_prob
+
+from mmdet.models.dense_heads.fcos_head import FCOSHead
+from mmdet.ops import ModulatedDeformConvPack
+from ..builder import HEADS
+
+
+@HEADS.register_module()
+class NASFCOSHead(FCOSHead):
+    """Anchor-free head used in `NASFCOS <https://arxiv.org/abs/1906.04423>
```

Comment:
```python
"""Anchor-free head used in NASFCOS <https://arxiv.org/abs/1906.04423>
```

Reply:
The old format is correct, which follows the rst syntax.

Reply:
Correct format should be 
```python
`NASFCOS <https://arxiv.org/abs/1906.04423>`_
```